src/main/java/core/basesyntax/figure/Main.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.figure;
+
+public class Main {
+    public static void main(String[] args) {
+
+        final int arraySize = 6;
+        Figure[] figures = new Figure[arraySize];
+        FigureSupplier figureSupplier = new FigureSupplier();
+
+        for (int i = 0; i < arraySize / 2; i++) {
+            figures[i] = figureSupplier.getRandomFigure();
+        }
+        for (int i = arraySize / 2; i < arraySize; i++) {
+            figures[i] = figureSupplier.getDefaultFigure();
```

Comment:
The way the loops are divided, half of the array will be filled with random figures, and the other half with default figures. If the arraySize is not even, the first half will have one less element than the second half. Consider using a different approach if you want the halves to be equal, or document this behavior if it is intentional.